src/serializer/visitors.js

```diff
@@ -103,7 +103,7 @@ function visitName(state, name, modified) {
   let ref = state.tryQuery(
     () => ResolveBinding(state.realm, name, doesNotMatter, state.val.$Environment),
     undefined,
-    true
```

Comment:
This is not an error collecting pass, so it makes no sense to report errors here.